semcore/dropdown-menu/src/DropdownMenu.jsx

```diff
@@ -521,6 +524,13 @@ function NestingTrigger(props) {
   );
 }
 
+function NestingItem(props) {
+  const { styles } = props;
+  const SDropdownNestingItem = Root;
+
+  return sstyled(styles)(<SDropdownNestingItem render={NestingTrigger} tabIndex={-1} />);
```

Comment:
maybe we should check that `NestingItem` has focusable children and only in that case set `tabIndex=-1`?

Reply:
For what? I suppose that it's always should be unfocusable